src/components/resumes/ResumeList/ResumeListAPI.js

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
```

Comment:
Move this getREsumeList function to this file https://github.com/fundev3/teamup-webapp/blob/develop/src/components/resumes/ResumesAPI.js